config/workflows/codeql.yml

```diff
@@ -0,0 +1,41 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master", "stable-[0-9][0-9]*" ]
+  pull_request:
+    branches: [ "master" ]
+  schedule:
+    - cron: '24 18 * * 3'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    strategy:
+      fail-fast: false
+      matrix:
+          language: [ 'java' ]
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v2
+      with:
+        languages: ${{ matrix.language }}
+    - name: Set up JDK 
+      uses: actions/setup-java@v2
+      with:
+        distribution: "temurin"
+        java-version: 11
+    - name: Assemble
+      run: |
+        mkdir -p $HOME/.gradle
+        echo "org.gradle.jvmargs=-Xmx2g -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryError" > $HOME/.gradle/gradle.properties
+        ./gradlew assembleDebug
```

Comment:
Shall we do something lik:
if [ $(./gradlew task --all | grep assembleGplay -c) -gt 0 ] ; then
./gradlew assembleGplayDebug
else
./gradlew assembleDebug
fi

As gplay pulls in google stuff and this might also have some warnings?

Reply:
`assembleDebug` assembles all debug variants (so for Files: huawei, gplay, versionDev and generic); this way we should have all of them covered